### ISSUE TITLE
Minimum TLS 1.2 fails for Azure Web App when TLS 1.3 is enabled

### ISSUE THREAD

**Issue by flexray** [toniblyx/prowler#5997]
### Steps to Reproduce

1. prowler azure --subscription-ids xx-88ae-4fe8-901a-16e33871e7c7 xx-5c28-4e32-94df-591a5baedf69 --az-cli-auth
2. Azure Web App with TLS 1.3 enabled

### Expected behavior

in file prowler/providers/azure/services/app/app_minimum_tls_version_12/app_minimum_tls_version_12.py

When checking for minimum TLS 1.2, please don't fail on TLS 1.3

### Actual Result with Screenshots or Logs

"Minimum TLS version is not set to 1.2 for app"

```
                if (
                    app.configurations
                    and getattr(app.configurations, "min_tls_version", "") == "1.2"
                ):
                    report.status = "PASS"
                    report.status_extended = f"Minimum TLS version is set to 1.2 for app '{app_name}' in subscription '{subscription_name}'."

```

### How did you install Prowler?

From pip package (pip install prowler)

### Environment Resource

Azure Cloud Shell

### OS used

Azure Cloud Shell

### Prowler version

4.5.3 

### Pip version

pip 23.0.1 from /usr/lib/python3.9/site-packages/pip (python 3.9)

**Comment by puchy22** [comment]
Hi @flexray, 

Thank you for the advice. I've fixed the check you mentioned in PR #6004. If possible, please test the changes and let me know if they work for you. 

Thank you for using Prowler! 🚀

**Comment by flexray** [comment]
Hi @puchy22,

Now it looks great "Minimum TLS version is set to 1.3 for app". Thank you!

Have a great day!
